news/2018-09-25-project-loved-one-year-week-of-september-23rd.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+title: "Project Loved: One Year / Week of September 23rd"
+date: 2018-09-25 20:00:00 +0000
+---
+
+[Project Loved](https://osu.ppy.sh/community/forums/120) is back again with a wide selection of beatmaps from all modes. Don't forget to cast your votes! We're also celebrating our first birthday today, so show us some love too!
```

Comment:
I would personally make the "big deal" of it being the birthday / anniversary first, rather than "oh and yeah we're a year old I guess" at the end.